gitlab/_backends/requests_backend.py

```diff
@@ -9,6 +9,11 @@
 from . import protocol
 
 
+class _StdoutStream:
+    def __call__(self, chunk: Any) -> None:
+        print(chunk)
```

Comment:
TBH I don't know why we have this. Just doing `action = print` and then `action(chunk)` would do in the function itself. Maybe the author had bigger plans for this.

Reply:
It worth adding it as a comment in the code so that contributors could peek it up and change it

Reply:
Thanks, I'll open a follow-up issue to track this. Maybe that's a bit cleaner than having comments in the code.

Reply:
Opened https://github.com/python-gitlab/python-gitlab/issues/2496